rpm_spec/gui-agent.spec.in

```diff
@@ -22,8 +22,8 @@
 #
 #
 
-
-# default value in case of no qubes-builder's one
+# Improve reproduciblity
+%undefine _annotated_build
```

Comment:
This is a bit unfortunate. If [this feature](https://fedoraproject.org/wiki/Changes/Annobin) breaks reproducibility, disabling it here (and in all other our packages?) fixes just this one package, but won't help us reproducing upstream Fedora packages for example. We need a solution that can be implemented upstream too. If this feature breaks reproducibility, I'm afraid we need to get which part exactly and make it reproducible, instead of disabling it. Asking people more familiar with this feature (on fedora devel ML?) may be a good idea.

Reply:
Yes at least I know that's part of the issue :)